src/components/Forms/FilterForm.jsx

```diff
@@ -1,14 +1,16 @@
-export default function FilterForm({types, filterChange, selectedType}) {
+export default function FilterForm({ types, filterChange, selectedType }) {
   return (
     <div>
       <h3>Filter Pokemon!</h3>
       <select
         value={selectedType}
         onChange={(event) => filterChange(event.target.value)}
       >
-        <option value='all'>All</option>
-        {types.map(({type}) => (
-          <option value={type}>{type}</option>
+        <option value="all">All</option>
+        {types.map(({ type }) => (
+          <option key={type} value={type}>
```

Comment:
There is one other element on this file that needs a key

Reply:
I've added a key to the option element on line 9 — not sure why React didn't bark at me about that in the console ...